lib/bolt/project.rb

```diff
@@ -49,7 +49,15 @@ def self.find_boltdir(dir)
 
     def initialize(path, type = 'option')
       @path = Pathname.new(path).expand_path
-      @config_file = @path + 'bolt.yaml'
+      @project_file = @path + 'bolt-project.yaml'
+
+      @warnings = []
+      if (@path + 'bolt.yaml').file? && project_file?
+        msg = "Found bolt.yaml and bolt-project.yaml in project directory '#{@path}', ignoring bolt.yaml"
```

Comment:
I didn't think of this during planning, but I'm a little concerned that anyone who has opted into an experimental feature could have their projects immediately break when this feature is released. Even though it's probably a relatively small subset of users who will have both files, I imagine it will cause some headaches to not have config loaded... Would it make sense to only ignore bolt.yaml if bolt-project.yaml has config in it?